app/views/qsets/show.html.erb

```diff
@@ -24,50 +24,58 @@
         <div class="panel-heading">
           <h4 class="panel-title">
             <div class="row valign">
-              <div class="col-sm-1">
-                <div class="net-votes"><p id="question-<%= question.id %>-net-votes"><%= question.plusminus %></p><br></div>
-              </div>
-              <div class="col-sm-1">
-                <div class="question-voting">
-                  <a class="upvote-button" data-vote="upvote" data-vote-qid="<%= question.id %>">Up</a><br>
-                  <a class="downvote-button" data-vote="downvote" data-vote-qid="<%= question.id %>">Down</a>
+              <div class="col-xs-9 col-sm-9">
+                <div class="col-xs-4 col-sm-3">
+                  <div class="net-votes"><p id="question-<%= question.id %>-net-votes"><%= question.plusminus %></p><br></div>
+                  <div class="question-voting">
+                    <!-- <a class="upvote-button" data-vote="upvote" data-vote-qid="<%= question.id %>">Up</a><br> -->
```

Comment:
Let's remove any comments that were for debugging, and leave in only the comments that clarify, organize, or warn about issues in the code.